command/agent/config/config.go

```diff
@@ -168,6 +173,12 @@ type TemplateConfig struct {
 	StaticSecretRenderInt    time.Duration `hcl:"-"`
 }
 
+type ExecConfig struct {
+	Command                []string  `hcl:"command,attr" mapstructure:"command"`
+	RestartOnSecretChanges string    `hcl:"restart_on_secret_changes,optional" mapstructure:"restart_on_secret_changes"`
+	RestartKillSignal      os.Signal `hcl:"-" mapstructure:"restart_kill_signal"`
```

Comment:
```suggestion
	RestartStopSignal      os.Signal `hcl:"-" mapstructure:"restart_stop_signal"`
```

per internal discussion